lsp/eslint.lua

```diff
@@ -142,7 +129,17 @@ return {
       }
 
       for _, file in ipairs(flat_config_files) do
-        if vim.fn.filereadable(root_dir .. '/' .. file) == 1 then
+        local found_files = vim.fn.globpath(root_dir, file, true, true)
+
+        -- Filter out files inside node_modules
+        local filtered_files = {}
+        for _, found_file in ipairs(found_files) do
+          if string.find(found_file, '/node_modules/') == nil then
+            table.insert(filtered_files, found_file)
```

Comment:
you could use `vim.fs.find()` (instead of `globpath()`) with a matcher function that skips node_modules files ? That would avoid this logic.

I wonder which one is faster. Performance matters if we are iterating node_modules ...